editoast/src/views/stdcm_search_environment.rs

```diff
@@ -99,8 +103,17 @@ impl From<StdcmSearchEnvironmentCreateForm> for Changeset<StdcmSearchEnvironment
 )]
 async fn overwrite(
     State(app_state): State<AppState>,
+    Extension(authorizer): AuthorizerExt,
     Json(form): Json<StdcmSearchEnvironmentCreateForm>,
 ) -> Result<impl IntoResponse> {
+    let authorized = authorizer
+        .check_roles([BuiltinRole::Stdcm].into())
```

Comment:
```suggestion
        .check_roles([BuiltinRole::Admin].into())
```

Envs will be overridden by a script. We don't want users to overwrite this data.

Reply:
Done.